lib/base/json.cpp

```diff
@@ -461,9 +453,9 @@ void JsonEncoder<prettyPrint>::AppendChars(Iterator begin, Iterator end)
 
 template<bool prettyPrint>
 inline
-void JsonEncoder<prettyPrint>::AppendJson(nlohmann::json json)
+void JsonEncoder<prettyPrint>::AppendJson(const nlohmann::json& json)
 {
-	nlohmann::detail::serializer<nlohmann::json>(nlohmann::detail::output_adapter<char>(m_Result), ' ').dump(std::move(json), prettyPrint, true, 0);
+	nlohmann::detail::serializer<nlohmann::json>(nlohmann::detail::output_adapter<char>(m_Result), ' ').dump(json, prettyPrint, true, 0);
 }
```

Comment:
I think with this change, https://github.com/Icinga/icinga2/blob/e097706d5b222b22ecad2a726f059a5cf2c0d54a/lib/base/json.cpp#L388 might do an additional copy.